### PR TITLE
Adding functions for setting response headers (set/header)

### PR DESCRIPTION
**Pull request by neptunao**

### CODE REVIEW

File: src/Express.re [hunk 1]
```diff
@@ -381,6 +381,8 @@ module Response = {
   [@bs.send.pipe: t]
   external redirectCode : (int, string) => complete = "redirect";
   [@bs.send.pipe: t] external redirect : string => complete = "redirect";
+  [@bs.send.pipe: t] external set : (string, string) => t = "set";
```
Comment (ncthbrt) [inline src/Express.re]: Don't these two functions do the same thing?
Would prefer a single function: `setHeader` or `header` maybe?

**Review by ncthbrt — COMMENTED** [review]
Happy to merge if we remove the redundant `set` function

**Comment by neptunao** [comment]
I've removed `header` function and renamed `set` to `setHeader`.

**Review by ncthbrt — APPROVED** [review]
Thanks for the PR. I'll merge it in and it'll be in the next release.

**Pull request merged**